src/contracts/pods/EigenPod.sol

```diff
@@ -182,86 +185,40 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
     }
 
     /**
-     * @notice  This function verifies that the withdrawal credentials of the podOwner are pointed to
+     * @notice  This function verifies that the withdrawal credentials for one or many validators of the podOwner that are pointed to
      * this contract. It also verifies the current (not effective) balance  of the validator.  It verifies the provided proof of the ETH validator against the beacon chain state
      * root, marks the validator as 'active' in EigenLayer, and credits the restaked ETH in Eigenlayer.
-     * @param oracleBlockNumber is the Beacon Chain blockNumber whose state root the `proof` will be proven against.
-     * @param validatorIndex is the index of the validator being proven, refer to consensus specs 
-     * @param proof is the bytes that prove the ETH validator's  withdrawal credentials against a beacon chain state root
+     * @param oracleTimestamp is the Beacon Chain timestamp whose state root the `proof` will be proven against.
+     * @param validatorIndices is a list of indices of the validators being proven, refer to consensus specs 
+     * @param proofs is the bytes that prove the ETH validator's  withdrawal credentials against a beacon chain state root
      * @param validatorFields are the fields of the "Validator Container", refer to consensus specs 
      * for details: https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#validator
      */
-    function verifyWithdrawalCredentialsAndBalance(
-        uint64 oracleBlockNumber,
-        uint40 validatorIndex,
-        bytes calldata proof,
-        bytes32[] calldata validatorFields
-    )
-        external
-        onlyWhenNotPaused(PAUSED_EIGENPODS_VERIFY_CREDENTIALS)
-        // check that the provided `oracleBlockNumber` is after the `mostRecentWithdrawalBlockNumber`
-        proofIsForValidBlockNumber(oracleBlockNumber)
+    function verifyWithdrawalCredentials(
+        uint64 oracleTimestamp,
+        uint40[] calldata validatorIndices,
```

Comment:
should we just make the `validatorIndices` a field on `WithdrawalCredentialProofs`

Reply:
I mean its not really a "proof" per say

Reply:
I approve G's idea. It's part of what you're proving, seems fine.

Reply:
would also be OK to include in `validatorFields`, IMO